packages/saltcorn-data/db/pg.js

```diff
@@ -182,7 +182,8 @@ const update = async (tbl, obj, id, opts = {}) => {
     .map(([k, v], ix) => `"${sqlsanitize(k)}"=$${ix + 1}`)
     .join();
   var valList = kvs.map(([k, v]) => v);
-  valList.push(id);
+  // TBD check that is correct - because in insert function opts.noid ? "*" : opts.pk_name || "id"
+  valList.push(id === "undefined"? obj[opts.pk_name]: id);
```

Comment:
I think this is good - but I would do: 

```
valList.push(id === "undefined"? obj[opts.pk_name||"id"]: id);
```

so "id" is the default primary key name

Reply:
I update code according your comments https://github.com/AleksandrSokolov/saltcorn/commit/7c1a59695920d5f5ad0b588a55a9e1492d51712f